### PR TITLE
Fix #11461

### LINKED ISSUE #11461: Migration tool. Create migrate with comma in defaultValue
The issue like that: https://github.com/yiisoft/yii2/issues/10969
If "defaultValue" contains comma, incorrect code will be generated. 

```
yii migrate/create add_field_to_table 
--fields="title:string(10):notNull:unique:defaultValue('te,st'),body:text:notNull"
```

### PR DESCRIPTION
**Pull request by pana1990**
| Q | A |
| --- | --- |
| Is bugfix? | yes |
| New feature? | no |
| Breaks BC? | no |
| Tests pass? | yes |
| Fixed issues | https://github.com/yiisoft/yii2/issues/11461 |

### CODE REVIEW

**Comment by pana1990** [comment]
not merge yet.

File: framework/console/Controller.php [hunk 1]
```diff
@@ -102,7 +102,7 @@ public function runAction($id, $params = [])
                 if (in_array($name, $options, true)) {
                     $default = $this->$name;
                     if (is_array($default)) {
-                        $this->$name = preg_split('/(?!\(\d+)\s*,\s*(?!\d+\))/', $value);
+                        $this->$name = preg_split('/(?!\((\' | ")*[^"\']+)\s*,\s*((?![^"\'(\)]*(\'|")*\)))/', $value);
```
Comment (Faryshta) [inline framework/console/Controller.php]: i am no good reading regex, what does this line is do? how is it different than before?

Comment (pana1990) [inline framework/console/Controller.php]: check : https://regex101.com/r/gZ5dH3/11

Comment (Faryshta) [inline framework/console/Controller.php]: again i am no good at regex, i don't know what I am seeing or how does it affect the commands.

In this case

What will do to

`yii fixture *, -User`

how about

`yii fixture -User -n="@common/fixtures"`

**Comment by pana1990** [comment]
@Faryshta @samdark @SilverFire up

**Comment by pana1990** [comment]
@samdark @SilverFire @cebe ready to merge

File: framework/CHANGELOG.md [hunk 2]
```diff
@@ -31,6 +32,12 @@ Yii Framework 2 Change Log
 - Bug #11561: Fixed DI container throwing exceptions for optional dependencies (SamMousa)
 - Bug #11571: Fixed `yii\db\ColumnSchemaBuilder` to work with custom column types (andrey-mokhov, silverfire)
 - Bug #11662: Fixed `schema-oci.sql` for RBAC (jonny7)
+- Bug #11686: `BaseArrayHelper::isIn()` comparison did not work in strict mode (taobig)
+- Bug #11723: Fixed PHP 7 + XDebug error handling displaying "Expected array for frame 0" (tanakahisateru)
+- Bug #11735: Fixed `yii\web\UploadedFile` to return `null` when there's no file uploaded (brummm)
+- Enh #8795: Refactored `yii\web\User::loginByCookie()` in order to make it easier to override (maine-mike, silverfire)
+- Enh #9948: `yii\rbac\PhpManager` now invalidates script file cache performed by 'OPCache' or 'APC' on file saving (klimov-paul)
+- Enh #11195: Added ability to append custom string to schema builder column definition (df2, samdark)
```
Comment (cebe) [inline framework/CHANGELOG.md]: something's wrong with the changelog here. can you rebase on master?

Comment (pana1990) [inline framework/CHANGELOG.md]: i think to resolve this issue in the last commit

Comment (pana1990) [inline framework/CHANGELOG.md]: isn't it?

**Comment by pana1990** [comment]
@cebe done!

**Comment by pana1990** [comment]
@SilverFire can you remove status:under development label?

**Comment by samdark** [comment]
Thank you!

**Pull request merged**